projects/rero/ng-core/src/lib/record/search/record-search-page.component.ts

```diff
@@ -86,7 +86,15 @@ export class RecordSearchComponent implements OnInit {
     itemHeaders?: any,
     aggregationsOrder?: Array<string>,
     aggregationsExpand?: Array<string>,
-    aggregationsBucketSize?: number
+    aggregationsBucketSize?: number,
+    pagination?: {
+      boundaryLinks?: boolean,
+      maxSize?: number,
+      previousText?: string,
+      nextText?: string,
+      firstText?: string,
+      lastText?: string
```

Comment:
For consistency between interfaces, it's better not to change by resource the texts of pagination buttons.